app/cli.py

```diff
@@ -124,7 +124,7 @@ def update_resource(resource, existing_resource):   # pragma: no cover
     existing_resource.languages = resource['languages']
 
 
-def reindex_all():
+def reindex_all():  # pragma: no cover
```

Comment:
I removed this from coverage since it is a cli command.

Reply:
Is it possible to just ignore this whole file from coverage if we're not worried about coverage for the cli?